client/app.jsx

```diff
@@ -31,7 +31,7 @@ function initializeGeoJSON(){
     type:'GET_GEOJSON'
   })
   
-  fetch('/api/trails?center=' + JSON.stringify(options.center.reverse()))
+  fetch('/api/trails?center=' + JSON.stringify([options.center[1], options.center[0]]))
```

Comment:
Yes

----- Reply message -----
From: "Nathan Bland" <notifications@github.com>
To: "NathanBland/trails" <trails@noreply.github.com>
Cc: "Joel" <jgimbel@live.com>, "Author" <author@noreply.github.com>
Subject: [NathanBland/trails] Set active in URL (#14)
Date: Mon, Jun 6, 2016 12:50 PM


In client/app.jsx<https://github.com/NathanBland/trails/pull/14#discussion_r65935674>:

> @@ -31,7 +31,7 @@ function initializeGeoJSON(){
>      type:'GET_GEOJSON'
>    })
>
> -  fetch('/api/trails?center=' + JSON.stringify(options.center.reverse()))
> +  fetch('/api/trails?center=' + JSON.stringify([options.center[1], options.center[0]]))


This is to stop the mutation correct?

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/NathanBland/trails/pull/14/files/47f669ed62a6e600c539eb52351a22d8ba054835#r65935674>, or mute the thread<https://github.com/notifications/unsubscribe/ABVaoKy_2qhfoltzHy7kTW7EF1PA8q3Kks5qJF3cgaJpZM4IvI_r>.